src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -100,6 +98,31 @@ public static IEnumerable<IEdmOperation> FindBoundOperations(this IEdmModel mode
             return FindAcrossModels(model, bindingType, findBoundOperations, mergeFunctions);  // search built-in EdmCoreModel and CoreVocabularyModel.
         }
 
+        /// Searches for bound operations based on the binding type, across models, returns an empty enumerable if no operation exists.
+        /// </summary>
+        /// <param name="model">The model to search.</param>
+        /// <param name="bindingType">Type of the binding.</param>
+        /// <returns>A set of operations that share the binding type or empty enumerable if no such operation exists.</returns>
+        public static IEnumerable<IEdmOperation> FindDeclaredBoundOperationsAcrossModels(this IEdmModel model, IEdmType bindingType)

```

Comment:
>FindDeclaredBoundOperationsAcrossModels [](http://example.com/codeflow?start=49&length=39)

For consistency, should be named "FindBoundOperationsAcrossModels"

Reply:
actually, most of these find across models, so perhaps just "findBoundOperations".  But, "FindBoundOperations" already exists -- what's the difference between the existing findboundoperations and this one?  How would I know when to use one versus the other?

Reply:
In particular, if this is just a more efficient FindBoundOperations, we should just change FindBoundOperations to have this improved implementation and not have two extension methods that do the same thing.